### DISCUSSION TITLE
Dashboard - Failed To Load Data

### DISCUSSION THREAD

**Discussion by whipzcfc** [vercel/community#5401] [Account Issue] [answered]
### Question

I have a personal account and a team account.  When logging into the UI, my personal account loads fine, but my team account does not load.  I receive a templated screen stating 'Error loading dashboard' and a snackbar alert that says 'Failed to load data, retrying..."
![Vercel](https://github.com/vercel/community/assets/124705630/fd3c3890-0a58-4eb7-ab44-e3173e37b87f)

**Comment by gilsonnunesfilho** [comment]
My personal account is having the same problem

**Comment by whipzcfc** [comment]
Support was unable to replicate the issue, but this resolved itself after approximately 20-30 minutes.

**Comment by mledwards** [comment]
Just got this on all projects

**Reply by amyegan** [reply]
Are you able to see your dashboard and projects now?

**Reply by mledwards** [reply]
Yes, it's all back. Thank you. Any idea why it happened? It didn't come up on the Vercel Status page.

**Reply by mledwards** [reply]
Got the error again :(

**Comment by ed1nh0** [comment]
I'm facing same issue here right now.
Earlier today I migrate a project to another team and everything went fine.
Came back right now and that error showed up.

**Reply by amyegan** [reply]
Thanks for reporting this. We are currently investigating. 

You can find the latest updates at https://www.vercel-status.com/incidents/7svxyp760r48

**Reply by ed1nh0** [reply]
Everything's fine now!

**Comment by swarnava** [comment, marked as answer]
This should be fixed now!

[a comment by an automated account is not shown]
